### PR TITLE
translation for 2021-04-05-ruby-2-5-9-released.md  in zh_tw

### PR DESCRIPTION
**Pull request by twlixin**
Translation for 2021-04-05-ruby-2-5-9-released.md  in zh_tw.

### CODE REVIEW

File: zh_tw/news/_posts/2021-04-05-ruby-2-5-9-released.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: news_post
+title: "Ruby 2.5.9 Released"
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2021-04-05-ruby-2-5-9-released.md:3]:
```suggestion
title: "Ruby 2.5.9 發布"
```

File: zh_tw/news/_posts/2021-04-05-ruby-2-5-9-released.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+author: "usa"
+translator: "twlixin"
+date: 2021-04-05 12:00:00 +0000
+lang: zh_tw
+---
+
+我們很高興宣布 Ruby 3.0.0 發布了。從 2015 年開始開發，目標是提高效能、支援並行性並提供型態檢查。Matz 表示「Ruby 3 將會比 Ruby 2 快 3 倍」，即 [Ruby 3x3](https://blog.heroku.com/ruby-3-by-3)。
```
Comment (JuanitoFatas) [inline zh_tw/news/_posts/2021-04-05-ruby-2-5-9-released.md:10]: Remove this line?

Comment (twlixin) [reply zh_tw/news/_posts/2021-04-05-ruby-2-5-9-released.md:10]: Sure.

**Comment by JuanitoFatas** [comment]
Thank you @twlixin 👏🏼

**Pull request closed without merging**